### PR TITLE
add TestRunner and remove TestBase in RecoveryServices

### PR DESCRIPTION
**Pull request by CaptainFanZzz**
## Description

add TestRunner and remove TestBase in RecoveryServices

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/RecoveryServices/RecoveryServices.Backup.Test/ScenarioTests/AzureSql/ContainerTests.cs [hunk 1]
```diff
@@ -13,39 +13,44 @@
...
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
 using Xunit;
 
 namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Test.ScenarioTests
 {
-    public partial class ContainerTests : RMTestBase
+    public partial class ContainerTests : RecoveryServicesBackupTestRunner
     {
-        public XunitTracingInterceptor _logger;
+        private readonly string _commonModule1 = $"ScenarioTests/Common.ps1";
```
Comment (vidai-msft) [inline src/RecoveryServices/RecoveryServices.Backup.Test/ScenarioTests/AzureSql/ContainerTests.cs:25]: DO NOT name with meaningless numbers. You can call it _containerCommModule.

**Review by vidai-msft — CHANGES_REQUESTED** [review]

**Comment by vidai-msft** [comment]
In the RecoveryServices.Backup.Test, it has lots of variables suffixed with numbers. Replace it with a meaningful name.

File: src/RecoveryServices/RecoveryServices.SiteRecovery.Test/ScenarioTests/RecoveryServicesSiteRecoveryTestRunner.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+namespace Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.Test.ScenarioTests
+{
+    public class RecoveryServicesSiteRecoveryTestRunner
+    {
+        protected readonly ITestRunner TestRunner;
+
+        protected RecoveryServicesSiteRecoveryTestRunner(ITestOutputHelper output)
+        {
+            TestRunner = TestManager.CreateInstance(output)
+                .WithCommonPsScripts(new[]
```
Comment (vidai-msft) [inline src/RecoveryServices/RecoveryServices.SiteRecovery.Test/ScenarioTests/RecoveryServicesSiteRecoveryTestRunner.cs:28]: Add one more statement .WithProjectSubfolderForTests("ScenarioTests") here.

**Review by vidai-msft — CHANGES_REQUESTED** [review]

**Comment by vidai-msft** [comment]
Make these changes so that the test runners overall have the same structure.

File: src/RecoveryServices/RecoveryServices.Backup.Test/ScenarioTests/AzureSql/ContainerTests.cs (hunk 1 above)
Comment (CaptainFanZzz) [reply src/RecoveryServices/RecoveryServices.Backup.Test/ScenarioTests/AzureSql/ContainerTests.cs]: Some names of these files are duplicate, so I update them to folder name. Thanks.

**Pull request merged**